services/github/github-hacktoberfest.service.js

```diff
@@ -0,0 +1,194 @@
+'use strict'
+
+const gql = require('graphql-tag')
+const Joi = require('@hapi/joi')
+const moment = require('moment')
+const { metric } = require('../text-formatters')
+const { nonNegativeInteger } = require('../validators')
+const { GithubAuthV4Service } = require('./github-auth-service')
+const {
+  documentation: githubDocumentation,
+  transformErrors,
+} = require('./github-helpers')
+
+const documentation = `
+  <p>
+    This badge is designed for projects hosted on GitHub which are
+    participating in
+    <a href="https://hacktoberfest.digitalocean.com">Hacktoberfest</a>,
+    an initiative to encourage participating in open-source projects. The
+    badge can be added to the project readme to encourage potential
+    contributors to review the suggested issues and to celebrate the
+    contributions that have already landed.
```

Comment:
```suggestion
    contributions that have already been submitted.
```

Landed sounds wrong as this isn't just considering merged PRs.

Reply:
Right-o. Gonna go with 'made' which I like slightly better.